### ISSUE TITLE
Critically Ill Patients Analysis and Prediction

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#466]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Critically Ill Patients Analysis and Prediction
:red_circle: **Aim** : Create a bunch of ML models to analyze the critically ill patients and predict their future.
:red_circle: **Dataset** : https://www.kaggle.com/datasets/margaritakholostova/support-ii-dataset-with-critically-ill-patients
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Shrutakeerti** [comment]
Hi, @abhisheks008! I would like to take up this issue.
Full name : Shrutakeerti Datta
GitHub Profile Link : https://github.com/Shrutakeerti
Participant ID (If not, then put NA) : N/A
Approach for this Project :  
1)  Exploratory Data Analysis (EDA):
Conducting the exploratory data analysis to understand the dataset's characteristics, identify patterns, and assess feature distributions. Analyzing the accuracy scores and additional metrics to identify the algorithm that performs best on the given dataset.
2)  Model Implementation: 
By choosing 3-4 machine learning algorithms as per said in the requirements suitable for the classification task (e.g., decision trees, random forests, support vector machines, and gradient boosting).
3)  Model Comparison:                                                                                                                                                             Calculating the  accuracy scores for each mode with  relevant metrics such as precision, recall, and F1 score for a more comprehensive evaluation.
4) Deployment and Testing : Analyzing the accuracy scores and additional metrics to identify the algorithm that performs best on the given dataset.

What is your participant role? IWOC 2024

**Comment by abhisheks008** [comment]
One issue at a time @Shrutakeerti

**Comment by Avdhesh-Varshney** [comment]
@abhisheks008 i would like to work on this issue. Could you please assign it to me?

Full name : Avdhesh Varshney
GitHub Profile Link : https://github.com/Avdhesh-Varshney
Participant ID (If not, then put NA) :
Approach for this Project :

- Analyzing the dataset.
- EDA processing.
- Filtering the dataset or performing feature engineering so that which can easily fit on the model.
- Feature selection using tree-based models, etc.
- Applying the different type of the models like, random forest, decision tree regression, xgboost, knn classifier, svm classifier, deep neural networks, etc.
- Summarizing all the algorithm's efficiency and scores at the end of the notebook.

What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.) IWOC

**Comment by aamiransari072** [comment]
Full name :  Md Aamir Ansari
GitHub Profile Link :
Participant ID (If not, then put NA) :
Approach for this Project : after examine the data set i found that data set colums are correctly not labeld so first labeld the data correctly then performing eda and find the most dependent feature for target and we have to predict multiple output so we have to both regression and calssification model and evalue them according to roc curve and confusion matrix 
What is your participant role? IWOC 2.O

**Comment by abhisheks008** [comment]
Issue assigned to you @Avdhesh-Varshney 

@aamiransari072 you can check out other issues present in this repo, as the issues are being assigned in a FCFS manner.
